src/pages/LandingPage.css

```
@@ -0,0 +1,300 @@
+  --tk-carbon-rgb: 201, 106, 111;
+  --tk-cross: #a8484d;
+  --tk-ink: #23324a;
+  --tk-ink-rgb: 35, 50, 74;
+  --tk-ink-soft: #4c5972;
+  --tk-graphite: #201c17;
+  --tk-graphite-soft: #5c5648;
+  --tk-line: #d8cbaa;
+  --tk-paid: #2e6b4c;
+  --tk-paid-rgb: 46, 107, 76;
+
+  --tk-font-mono: 'IBM Plex Mono', 'SFMono-Regular', ui-monospace, Menlo, monospace;
```

Stylelint (17.14.1)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)